exercises/react/react_spec.cr

```diff
@@ -0,0 +1,124 @@
+require "spec"
+require "./react"
+
+describe React::InputCell do
+  it "have a value" do
+    input = React::InputCell.new(10)
+    input.value.should eq 10
+  end
+
+  it "can have values set" do
+    input = React::InputCell.new(4)
+    input.value = 20
+    input.value.should eq 20
+  end
+end
+
+describe React::ComputeCell do
+  it "calculate initial value" do
+    input = React::InputCell.new(1)
+    output = React::ComputeCell.new(input) { |x| x + 1 }
```

Comment:
I was inconsistent with my names, sometimes using `x` and sometimes using `v`. Please let me fix that